### DISCUSSION TITLE
binding multiple host IPs to service load balancers

### DISCUSSION THREAD

**Discussion by jrwren** [k3s-io/k3s#10060] [Q&A]
I have k3s on a single node as my own personal little test cluster.

I want to test running 2 ingress, istio and nginxinc.

I've added an addition IP the host. Is there a way to tell ServiceLB that I now have 2 IP addresses it can use? It has already assigned one to the istio ingress (although it might be binding * instead of the specific host IP).

My guess is that I need to set the hostips somehow, somewhere, but I have not found the right docs.

**Comment by brandond** [comment]
If both services use the same ports, they cannot run on the same hosts. You need to use distinct ports for LoadBalancer services when using ServiceLB, if you want them to run on all nodes.

Ref: https://docs.k3s.io/networking/networking-services#how-servicelb-works

**Comment by mcarbonneaux** [comment]
there no solution to use multiple ip of the host with ServiceLB ?

for example i need to have traffic come from one network to go to specific k3s/k8s service and other traffic from another network to come to another k8s services.
